templates/kubernetes/deploy/ingress.yml

```diff
@@ -10,8 +10,68 @@ metadata:
     ingress.kubernetes.io/ssl-redirect: "true"
     cert-manager.io/cluster-issuer: clusterissuer-letsencrypt-production
     # CORS
-    nginx.ingress.kubernetes.io/enable-cors: "true"
```

Comment:
Enabling CORS is now handled by the configuration snippet, leaving it here adds the `Access-Control-Allow-Origin` header twice.